test/runtime/connectivity.go

```diff
@@ -656,8 +658,10 @@ var _ = Describe("RuntimeValidatedConntrackTest", func() {
 
 		for _, endpointToConfigure := range endpointsToConfigure {
 			// ConntrackLocal must be disabled as it depends on Conntrack
-			vm.SetAndWaitForEndpointConfiguration(endpointToConfigure, helpers.OptionConntrackLocal, helpers.OptionDisabled)
-			vm.SetAndWaitForEndpointConfiguration(endpointToConfigure, helpers.OptionConntrack, helpers.OptionDisabled)
+			err := vm.SetAndWaitForEndpointConfiguration(endpointToConfigure, helpers.OptionConntrackLocal, helpers.OptionDisabled)
+			Expect(err).To(BeNil(), "Cannot disable ConntrackLocal for the endpoint %q", endpointToConfigure)
+			err = vm.SetAndWaitForEndpointConfiguration(endpointToConfigure, helpers.OptionConntrack, helpers.OptionDisabled)
+			Expect(err).To(BeNil(), "Cannot disable Conntrackl for the endpoint %q", endpointToConfigure)
```

Comment:
Please be more careful about typing in log messages. 
`Conntrackl` --> `ConnTrackLocal`.